internal/server/rest.go

```diff
@@ -141,6 +141,7 @@ func createRestServer(ip string, port int, needToken bool) *http.Server {
 
 	r := mux.NewRouter()
 	r.HandleFunc("/", rootHandler).Methods(http.MethodGet, http.MethodPost)
+	r.HandleFunc("/system/status", getMetricsHandler).Methods(http.MethodGet)
```

Comment:
How about merge to rootHandler? Root handler also return elapse time which is also a status, so I prefer to not create a new API.